tokio/src/runtime/mod.rs

```diff
@@ -19,7 +19,7 @@
 //! Creating a [`Runtime`] does the following:
 //!
 //! * Spawn a background thread running a [`Reactor`] instance.
-//! * Start a [`ThreadPool`] for executing futures.
+//! * Start a `ThreadPool` for executing futures.
```

Comment:
Why the removal of this link? Also, can't we just rely on auto-doc-links, and not give the absolute URL?

Reply:
`ThreadPool` is going to go away in a follow up PR.

Reply:
@carllerche if the `ThreadPool` type is going away, why not change this to 
```suggestion
//! * Start a thread pool for executing futures.
```